code/game/objects/items/choice_beacon.dm

```diff
@@ -95,9 +95,18 @@
 			/obj/item/instrument/recorder,
 			/obj/item/instrument/harmonica,
 			/obj/item/instrument/piano_synth/headphones,
+			// BANDASTATION ADDITION BEGIN - Expand instruments
+			/obj/structure/musician/drumskit,
+			/obj/item/instrument/soundhand_bass_guitar,
+			/obj/item/instrument/soundhand_rock_guitar,
+			// BANDASTATION ADDITION END - Expand  instruments
 		)
-		for(var/obj/item/instrument/instrument as anything in possible_instruments)
-			instruments[initial(instrument.name)] = instrument
+		// BANDASTATION EDITION BEGIN - Expand  instruments
+		// for(var/obj/item/instrument/instrument as anything in possible_instruments)
+		// 	instruments[initial(instrument.name)] = instrument
+		for(var/atom/instrument_path as anything in possible_instruments)
+			instruments[initial(instrument_path.name)] = instrument_path
+		// BANDASTATION EDITION END - Expand  instruments
```

Comment:
Не нужно лишних строк, да и принимать atom тоже незачем я думаю
```suggestion
		for(var/obj/instrument_path as anything in possible_instruments) // BANDASTATION EDIT - Expand instruments (Allows any obj instead of instrument)
			instruments[initial(instrument_path.name)] = instrument_path
```